### PR TITLE
Add more implemented features to README.md

### PR DESCRIPTION
**Pull request by StaticallyTypedRice**
I noticed that the feature list in the README is missing a few new features, so I took the liberty to add my favorite features that were not included!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,19 +55,25 @@ Made with [Rust](https://www.rust-lang.org), [Actix](https://actix.rs/), [Infern
...
+- Only a minimum of a username and password is required to sign up!
+- User avatar support.
 - Clean, mobile-friendly interface.
   - Live-updating Comment threads.
   - Full vote scores `(+/-)` like old reddit.
   - Themes, including light, dark, and solarized.
   - Emojis with autocomplete support. Start typing `:`
   - User tagging using `@`, Community tagging using `#`.
+  - Integrated image uploading in both posts and comments.
+  - A post can consist of a title and any comination of self text, a URL, or nothing else.
```
Comment (Nutomic) [inline README.md]: combination

Comment (StaticallyTypedRice) [reply README.md]: Sorry, I'm not sure what you mean. Not familiar with Git/GitHub lingo yet.

Comment (Nutomic) [reply README.md]: just saying you have a typo in there, `comination` ;)

Comment (StaticallyTypedRice) [reply README.md]: Oh, I'll fix that right now!

**Comment by dessalines** [comment]
Looks good, I'll merge this with a few small changes, thanks!

**Comment by StaticallyTypedRice** [comment]
I've also added a list of official repository mirrors to the README in order to help contributors use their preferred platform, per my musings [in this Mastodon post](https://mastodon.social/web/statuses/103541362770478711). I could find three, but if there are more then please link them so I can add them!

**Pull request merged**